tests/data/transformer-module/06-module-with-syntax-error/module/incomplete.fga

```diff
@@ -0,0 +1,3 @@
+module
```

Comment:
Could we maybe put this under `dsl-syntax-validation-cases.yaml` rather than the module transform tests? (I'm guessing that would then require the fix to be implemented across Go and Java too)

Reply:
Currently Java & Go listeners dont catch on a missing module names;

https://github.com/openfga/language/blob/e4a4d6ab4e1c87a8e6f93301605f8b485ee4baac/pkg/java/src/main/java/dev/openfga/language/OpenFgaDslListener.java#L67-L69

https://github.com/openfga/language/blob/f45e80a9f169370f7cbeb5da9d147e5ff0e7305e/pkg/go/transformer/dsltojson.go#L103-L109

Reply:
Nice!